### DISCUSSION TITLE
Невозможность использования API на старых девайсах

### DISCUSSION THREAD

**Discussion by E2r7hN07Fl47** [OpenVK/openvk#532] [Ideas] [answered]
Предлагаю сделать официальное HTTP зеркало API OpenVK, чтобы можно было его использовать на старых девайсах, где нет поддержки TLSv1 и/или нет новых сертификатов.

**Comment by NoPlagiarism** [comment, marked as answer]
Добавлено 18 апреля
http://openvk.co/
